testproject/Packages/manifest.json

```diff
@@ -1,15 +1,15 @@
 {
   "dependencies": {
-    "com.unity.addressables": "1.16.19",
-    "com.unity.collab-proxy": "1.9.0",
-    "com.unity.ide.rider": "3.0.7",
-    "com.unity.ide.visualstudio": "2.0.11",
-    "com.unity.ide.vscode": "1.2.3",
-    "com.unity.mathematics": "1.2.1",
+    "com.unity.addressables": "1.18.19",
```

Comment:
Are the manifest and packages json file modifications part of this PR?

Reply:
Yes, they are intended to be here. :)